oslo/torch/distributed/parallel_context.py

```diff
@@ -316,6 +325,7 @@ def from_openmpi(
             tensor_parallel_size (int): tensor parallel size
             tensor_parallel_depth (Optional[int]): tensor depth for tensor 2.5 parallelism
             tensor_parallel_mode (Optional[ParallelMode]): tensor parallel mode
+            memory_priority (bool): Whether or not to use memory priority mode (if False, speed priority mode)
```

Comment:
Should we add this to parallel context?
How about adding to TensorParallel.

Reply:
@hyunwoongko Since we need to parallel_context.memory_priority at ColLinear1D, I think it needs to be in parallel_context.

Reply:
We can add this using `setattr` no?

Reply:
I don't think so. Since the user would be only using parallel_context or OsloParallel (not each TensorParallel, PipelineParallel etc.) it needs to be in parallel_context not TensorParallel. How do you think?